lib/metasploit/framework/login_scanner/mssql.rb

```diff
@@ -66,7 +68,8 @@ def attempt_login(credential)
           }
 
           begin
-            if mssql_login(credential.public, credential.private, '', credential.realm)
+            client = Rex::Proto::MSSQL::Client.new(framework_module, framework, host, port)
```

Comment:
you'll want to make sure that the cleanup method is being called on this client, otherwise you'll leak sockets